it/test/uk/gov/hmrc/agentclientrelationships/controllers/RemoveAuthorisationControllerISpec.scala

```diff
@@ -321,6 +321,39 @@ trait RemoveAuthorisationControllerISpec
 
     }
 
+    "return 204 when PartialAuth exists in PartialAuth Repo and not not exists in InvitationRepo" in new StubsForThisScenario {
+      val newInvitation: Invitation = Invitation
+        .createNew(arn.value, service, nino, nino, "TestClientName", expiryDate, None)
+        .copy(status = PartialAuth)
+
+      await(
```

Comment:
I don't think we need to create an invalid delete record to prove that one does not exist (I say invalid because it is being created with a nino) - the verifyDeleteRecordNotExists assertion should always be true in these cases